### DISCUSSION TITLE
Chroma Control no longer seeing rgb fans via motherboard header Asus

### DISCUSSION THREAD

**Discussion by CHRIS00711** [ChromaControl/ChromaControl#330] [Q&A] [answered]
So I updated bios and now the fans connected to rgb headers on the motherboard no longer work via chroma control. the fans still work using armory crate so I know the motherboard see them but im guessing chroma control no longer is. Chroma control still working with icue fans and razer keyboard and mouse.

**Comment by schaubadahut** [comment]
I have the same issue. I saw another thread where the recent update for Armoury Crate isn't playing nice with ChromaControl and it's currently being worked on. I don't know if the BIOS update would have anything to do with it though.

**Reply by CHRIS00711** [reply]
hmm is there a place to get a older version of armory crate I wonder.

**Reply by schaubadahut** [reply]
You might see if there isn't a way to "roll back" Armoury Crate to undo the update.

**Comment by andrewbabbittdev** [comment, marked as answer]
This has been talked about heavily in the Discord server, Armory Crates AURA SDK is broken and has been for a while now. You must uninstall with the uninstall tool, then use the older full installation package dated in April. **YOU MUST NOT UPDATE ARMORY CRATE AFTER INSTALLING THIS**

[Armoury_Crate_Uninstall_Tool.zip](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Uninstall_Tool.zip?model=Armoury%20Crate)

[Armoury_Crate_Full_Installation_Package.zip](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Full_Installation_Package.zip?model=Armoury%20Crate)

**Reply by lothar-cell** [reply]
> This has been talked about heavily in the Discord server, Armory Crates AURA SDK is broken and has been for a while now. You must uninstall with the uninstall tool, then use the older full installation package dated in April. **YOU MUST NOT UPDATE ARMORY CRATE AFTER INSTALLING THIS**
> 
> [Armoury_Crate_Uninstall_Tool.zip](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Uninstall_Tool.zip?model=Armoury%20Crate)
> 
> [Armoury_Crate_Full_Installation_Package.zip](https://dlcdnets.asus.com/pub/ASUS/mb/14Utilities/Armoury_Crate_Full_Installation_Package.zip?model=Armoury%20Crate)

Is this still applicable?

**Reply by andrewbabbittdev** [reply]
> Is this still applicable?

I don't know the state of armory crate support, from what I have been told it may have been fixed in recent versions but I don't have 100% confirmation. I no longer have an ASUS MB and we have moved onto using OpenRGB in dev builds.

**Reply by lothar-cell** [reply]
Thank you, does it mean the dev build needs OpenRGB installed?

**Reply by andrewbabbittdev** [reply]
> Thank you, does it mean the dev build needs OpenRGB installed?

The dev build includes a copy of OpenRGB optimized for use with Chroma Control.
